### ISSUE TITLE
Build: Make sure we use Node LTS version throughout scripts and containers 

### ISSUE THREAD

**Issue by hugohaggmark** [grafana/grafana#17720]
**What would you like to be added**:
Today there are a couple of places you need to change when bumping the Node version. We should look into a way to have one place of truth for the supported Node version.  

**Why is this needed**:
- We've had some issues where CircleCI uses an older Node version and tests fail because of this. 
- We've had other issues where someone is using a different Node version and Grafana doesn't work as expected.

**Comment by hugohaggmark** [comment]
I've looked into this and I found the following:
- CircleCI doesn't support reading `engines` from package.json
- CircleCI doesn't support image names to as an environment variable
- Doesn't seems to be a way to specify Node version in DockerFile as an environment variable

This means that we would have to construct something ourselves that reads `engines` version from `package.json` or some other file like `.node-version` and then during (for instance) `pre-commit` validate that the versions entered in the different places as described in [UPGRADING_DEPENDENCIES.md](https://github.com/grafana/grafana/blob/master/UPGRADING_DEPENDENCIES.md) are valid.

Don't know if the effort for this and its future maintenance is justifiable. WDYT?
@xlson @marefr @dprokop @torkelo
